test/osb_test/osb_test.go

```diff
@@ -445,6 +446,24 @@ var _ = Describe("Service Manager OSB API", func() {
 		})
 	})
 
+	Describe("Post Binding Adapt Credentials", func() {
```

Comment:
Would it make sense to add the other cases as well? Currently we have call to working broker and call to broken broker. We might add the cases for verifying query params are preserved as well as calls to missing broker and stopped broker.

Reply:
@KirilKabakchiev Could you check if the added tests are sufficient from your point of view?

Reply:
looks good, thanks